index.html

```diff
@@ -498,22 +501,30 @@ <h2>Start an idle period algorithm</h2>
           <p class='note'>This is intended to allow user agents to delay the
           start of idle periods as needed to optimise the power usage of the
           device. For example, if the <code>Document</code>'s <a>hidden</a>
-          attribute ([[!page-visibility]]) is <code>true</code> then the user agent can
-          throttle idle period generation, for example limiting the Document to
-          one idle period every 10 seconds to optimize for power usage.</p>
+          attribute ([[!page-visibility]]) is <code>true</code> then the user
+          agent can throttle idle period generation, for example limiting the
+          Document to one idle period every 10 seconds to optimize for power
+          usage.</p>
         </li>
         <li>Let <var>now</var> be the current time.</li>
-        <li>Let <var>deadline</var> be a time in the future until which the
-        browser expects to remain idle. The user agent SHOULD choose
-        <var>deadline</var> to ensure that no time-critical tasks will be
-        delayed even if a callback runs for the whole time period from
-        <var>now</var> to <var>deadline</var>. As such, it should be set to the
-        minimum of: the closest timeout in the <a>list of active timers</a> as
-        set via <a>setTimeout</a> and <a>setInterval</a>; the scheduled
-        runtime for pending animation callbacks posted via
-        <a>requestAnimationFrame</a>; pending internal timeouts such
-        as deadlines to start rendering the next frame, process audio or any
-        other internal task the user agent deems important.</li>
+        <li>Let <var>deadline</var> be a time in the future until which the 
+        UA expects the <var>window</var>'s <a>window event loop</a> is likely
+        to remain idle. The user agent SHOULD choose <var>deadline</var> to 
+        ensure that no time-critical tasks will be delayed even if a callback 
```

Comment:
Again, what does "time-critical tasks" mean? That's not a thing defined anywhere in this or any other spec. We can't have a normative text like that.